### PR TITLE
add new android operation(open app,exit,stop) 

### PR DESCRIPTION
**Pull request by kithib**
**Features**

- xx
- yy
    
**Feature Docs**

**Influence**

You can use EXIT to return to the main interface of the mobile phone and turn off the application that is running in the background
You can use stop to return to the task success information
You can use OpenApp without XML to turn on the mobile app 
You can use Click Text without XML to click special text icon
**Result**
截屏2024-04-18 11 57 39

**Other**

截屏2024-04-18 11 57 19
截屏2024-04-18 15 30 13
截屏2024-04-18 16 03 14

### CODE REVIEW

File: metagpt/environment/android/android_ext_env.py [hunk 1]
```diff
@@ -224,7 +259,98 @@ def user_swipe(self, x: int, y: int, orient: str = "up", dist: str = "medium", i
...
         return swipe_res
+
+    @mark_as_writeable
+    def user_exit(self) -> str:
+        adb_cmd = f"{self.adb_prefix_shell} am start -a android.intent.action.MAIN -c android.intent.category.HOME"
+        exit_res = self.execute_adb_with_cmd(adb_cmd)
+        return exit_res
+
+    def _ocr_text(self, text: str) -> list:
+        if not self.screenshot_dir.exists():
```
Comment (better629) [inline metagpt/environment/android/android_ext_env.py:274]: already use `self.create_device_path`. no need.

File: metagpt/environment/android/android_ext_env.py [hunk 2]
```diff
@@ -224,7 +259,98 @@ def user_swipe(self, x: int, y: int, orient: str = "up", dist: str = "medium", i
...
+            logger.info(
+                f"Failed to execute action click text ({text}). There are too many text \"{text}\" in the screenshot.")
+
+    @mark_as_writeable
+    def user_stop(self):
+        logger.info("Successful execution of tasks")
+
+    @mark_as_writeable
+    def user_click_icon(self, icon_shape_color: str) -> str:
+        if not self.screenshot_dir.exists():
```
Comment (better629) [inline metagpt/environment/android/android_ext_env.py:325]: same.

**Comment by better629** [comment]
lgtm

**Review by geekan — APPROVED** [review]

**Pull request merged**